tools/run_tests/jobset.py

```diff
@@ -254,14 +254,21 @@ def stdout(self=self):
       stdout = self._tempfile.read()
       self.result.message = stdout[-_MAX_RESULT_SIZE:]
       return stdout
+    is_http2 = False
+    msg = ''
+    if 'http2' in self._spec.shortname:
```

Comment:
I'd still prefer a correct fix for this.

There should be no reason to read stdout until process termination.

This code is executed whenever we poll the process status.

It would be better to move this block into one of the if blocks below.